drivers/xen/gnttab.c

```diff
@@ -311,7 +319,6 @@ int gnttab_unmap_refs(struct gnttab_map_grant_ref *unmap_ops, unsigned int count
 	return HYPERVISOR_grant_table_op(GNTTABOP_unmap_grant_ref, unmap_ops, count);
 }
 
-
```

Comment:
I'd prefer to see formatting changes in a separate commit.

Reply:
Done